### ISSUE TITLE
Zoey/Tomster should have `alt=""`

### ISSUE THREAD

**Issue by chancancode** [ember-learn/guides-source#1124]
@MelSumner brought it to my attention.

According to [the Web Accessibility tutorial](https://www.w3.org/WAI/tutorials/images/decorative/), the Zoey/Tomster images in "Zoey/Tomster says..." are decorative, therefore should have an empty `alt` attribute (`alt=""`).

I addressed it in the tutorial in https://github.com/ember-learn/super-rentals-tutorial/pull/74, but this problem is more widespread than just the tutorial.
